docs/Developers/Newcomers_Guide.rst

```diff
@@ -164,6 +164,14 @@ git/GitHub, we strongly recommend following a tutorial, such as `this one
 We also have a `page dedicated to git commands <http://coala.io/git>`_ that
 will help you learn the basics.
 
+.. note::
+
+    You can view some of our repositories on
+    `Gitlab <https://gitlab.com/coala>`_.
+    Gitlab contributions also count to the promotion of a
```

Comment:
I think the promotion aspect fits more at the beginning where the promotion is first mentioned.